client/packages/coldchain/src/Equipment/ImportAsset/UploadTab.tsx

```diff
@@ -155,11 +155,6 @@ function getImportHelpers<T, P>(
     const prop = t(localeKey) as keyof P;
     const value = row[prop] ?? '';
     if (value === undefined || (value as string).trim() === '') {
-      rowErrors.push(
```

Comment:
Thoughts here please!

Removes mandatory requirement of catalogue item code, and store code if on central... docs say these things are required, but when you create individually via the modal, you don't need to assign a store nor create from catalogue... so why do we mandate it here?

Reply:
we did have the store code mandatory on central, though that has changed and we now have 'unassigned' equipment, so the store change is valid.

for the catalogue item - we'd require a type to be assigned, if the catalogue item isn't used.

( the allowance of non-catalogue items was another change part way through development, hence the functional mismatch here )